config/windows/build.xml

```diff
@@ -90,7 +90,7 @@
 		<mkdir dir="${bin.native}/build"/>
 
 		<!-- CORE -->
-		<build name="lwjgl" dest="${bin.native}/core">
+		<build name="lwjgl" dest="${bin.native}/core" flags="/wd4710">
```

Comment:
Oh, that's been a bad idea then. The problem is that this warning is reported by the linker(!), because of link time code generation I suppose, and so far I didn't find a nice solution which works and is limited to this one function or compilation unit.